cmake/Modules/FindSecurity.cmake

```diff
@@ -1,9 +1,32 @@
-IF (SECURITY_INCLUDE_DIR AND SECURITY_DIRS)
-  SET(SECURITY_FOUND TRUE)
+# Find Security.framework
+# This will define :
+#
+# SECURITY_FOUND
+# SECURITY_LIBRARIES
+# SECURITY_LDFLAGS
+# SECURITY_HAS_SSLCREATECONTEXT
+#
+
+IF (SECURITY_INCLUDE_DIR AND SECURITY_LIBRARIES)
+	SET(SECURITY_FOUND TRUE)
 ELSE ()
-  FIND_PATH(SECURITY_INCLUDE_DIR NAMES Security/Security.h)
-  FIND_LIBRARY(SECURITY_DIRS NAMES Security)
-  IF (SECURITY_INCLUDE_DIR AND SECURITY_DIRS)
-    SET(SECURITY_FOUND TRUE)
-  ENDIF ()
+	FIND_PATH(SECURITY_INCLUDE_DIR NAMES Security/Security.h)
+	FIND_LIBRARY(SECURITY_LIBRARIES NAMES Security)
+	IF (SECURITY_INCLUDE_DIR AND SECURITY_LIBRARIES)
+		IF (NOT Security_FIND_QUIETLY)
+			MESSAGE("-- Found Security ${SECURITY_LIBRARIES}")
+		ENDIF()
+		SET(SECURITY_FOUND TRUE)
+		SET(SECURITY_LDFLAGS "-framework Security")
```

Comment:
Same as for the other module changes. Here, both SECURITY_LDFLAGS and SECURITY_HAS_SSLCREATECONTEXT may be missing in the first case.